index.html

```diff
@@ -0,0 +1,1305 @@
+            alt="newsweek-img">
+
+          <div class="card-body pl-0 ">
+            <div class="card-title">
+              <h4>Easing of Coronavirus Quarantine Measures in Sacramento Leaves Experts Torn</h4>
+            </div>
+            <div class="card-text">
+              <p>More than 760 people in the U.S. have been diagnosed with COVID-19.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-3 col-md-6 col-sm-12 col-xm-12 w-100">
+        <div class="card mt-5 w-100 ">
+          <img class="img-fluid w-100"
+            src="https://d.newsweek.com/en/full/1572746/italy-coronavirus.jpg?w=397&amp;h=295&amp;f=d383ecb783db9c2da5247764a5b4bdad"
+            alt="newsweek-img">
+
+          <div class="card-body pl-0 ">
+            <div class="card-title">
+              <h4>Easing of Coronavirus Quarantine Measures in Sacramento Leaves Experts Torn</h4>
+            </div>
+            <div class="card-text">
+              <p>More than 760 people in the U.S. have been diagnosed with COVID-19.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-3 col-md-6 col-sm-12 col-xm-12 w-100">
+        <div class="card mt-5 w-100 ">
+          <img class="img-fluid w-100"
+            src="https://d.newsweek.com/en/full/1572653/donald-trump-taliban-afghanistan-peace-deal-fleeced.jpg?w=397&amp;h=295&amp;f=9513e4596a7b771916e1a05c6d29e311"
+            alt="newsweek-img">
+          <div class="card-body pl-0 ">
+            <div class="card-title">
+              <h4>Trump May Have Been 'Fleeced' in Taliban Peace Deal, Senator Says</h4>
+            </div>
+            <div class="card-text">
+              <p>More than 760 people in the U.S. have been diagnosed with COVID-19.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-3 col-md-6 col-sm-12 col-xm-12 w-100">
+        <div class="card mt-5 w-100 ">
+          <img class="img-fluid w-100"
+            src="https://d.newsweek.com/en/full/1572746/italy-coronavirus.jpg?w=397&amp;h=295&amp;f=d383ecb783db9c2da5247764a5b4bdad"
+            alt="newsweek-img">
+          <div class="card-body pl-0 ">
+            <div class="card-title">
+              <h4>Doctor in Coronavirus-Stricken Italy: 'Every Ventilator Becomes Like Gold'</h4>
+            </div>
+            <div class="card-text">
+              <p>More than 760 people in the U.S. have been diagnosed with COVID-19.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="row  pl-lg-4 pr-lg-4 bg-white pt-sm-3">
+      <div class="col-12 mt-4">
+        <div class="d-flex magazine ">
+          <h5 class="m-auto d-flex pb-2"><span class="icon-n mr-3"></span>FEATURED SLIDESHOWS</h5>
+        </div>
+      </div>
+    </div>
```

Comment:
The article images in this section don't include the badge with the number of images. Please include it. Also, please make sure that when you fix the body, the images in this section reflect those changes, too.

Yours:
![Screenshot from 2020-03-14 18-26-08](https://user-images.githubusercontent.com/51926937/76692825-4f620800-6621-11ea-80c8-9b08be627b79.png)

Original:
![Screenshot from 2020-03-14 18-26-15](https://user-images.githubusercontent.com/51926937/76692826-525cf880-6621-11ea-8cac-cdb39bfea4ec.png)